packages/docs/src/routes/design/ev-interface.txt

```diff
@@ -0,0 +1,7 @@
+interface IEventHandler {
+    component: MaterialComponent;
```

Comment:
Component or MDComponent

Reply:
Hmm I guess I meant `MDComponent`

Reply:
No I actually meant `MaterialComponent` the super class of all our components. Improved the docs for that